commitlintplugins.test.ts

```diff
@@ -73,6 +73,33 @@ test('body-max-line-length6', () => {
 });
 
 
+test('prefer-slash-over-backslash1', () => {
+    let commitMsgWithBackslash = "foo\\bar: bla bla bla";
+    let preferSlashOverBackslash1 = runCommitLintOnMsg(commitMsgWithBackslash);
+    expect(preferSlashOverBackslash1.status).not.toBe(0);
+});
+
+test('prefer-slash-over-backslash2', () => {
+    let commitMsgWithSlash = "foo/bar: bla bla bla";
+    let preferSlashOverBackslash2 = runCommitLintOnMsg(commitMsgWithSlash);
+    expect(preferSlashOverBackslash2.status).toBe(0);
+});
+
+
+test('prefer-slash-over-backslash3', () => {
+    let commitMsgWithSlash = "/bar: bla bla bla";
+    let preferSlashOverBackslash3 = runCommitLintOnMsg(commitMsgWithSlash);
+    expect(preferSlashOverBackslash3.status).toBe(0);
+});
+
+
+test('prefer-slash-over-backslash4', () => {
+    let commitMsgWithBackslash = "\\bar: bla bla bla";
+    let preferSlashOverBackslash4 = runCommitLintOnMsg(commitMsgWithBackslash);
+    expect(preferSlashOverBackslash4.status).not.toBe(0);
+});
```

Comment:
@realmarv not sure I see the value of the last 2 tests, let's just remove them because they add more confusion than anything